packages/cli/src/storybook/getProjectWebpackConfig.ts

```diff
@@ -29,14 +29,11 @@ async function getWebpackConfig(loadOptions) {
   return replaceWebpackRules(webpackConfig, replaceDefaultMediaLoader);
 }
 
-async function getStorybookReactWebpackConfig() {
-  return getWebpackConfig(require('@storybook/react/dist/server/options').default);
-}
-
 export async function getStorybookProjectWebpackConfig(): Promise<webpack.Configuration | void> {
-  if (storybookIsInstalled(StorybookFramework.REACT)) {
-    /** @TODO (maybe) we need to have sep routines for different versions but lets not get ahead of ourselves */
-    return getStorybookReactWebpackConfig();
+  /** @TODO (maybe) we need to have sep routines for different versions but lets not get ahead of ourselves */
```

Comment:
rm comment, not needed here then anymore

Reply:
done